src/it/scala/net/snowflake/spark/snowflake/SnowflakeResultSetRDDSuite.scala

```diff
@@ -1193,6 +1193,17 @@ class SnowflakeResultSetRDDSuite extends IntegrationSuiteBase {
     assert(count == LARGE_TABLE_ROW_COUNT)
   }
 
+  test("count super large table") {
```

Comment:
@dohongdayi Thanks a lot for your PR on spark connector.
`getRowCount()` is a test function to run a query, It is not related to Spark Connector production code.
So, I don't think this test reproduced the issue.

Reply:
Sure, please review again.

Reply:
This case can succeed without any fix.

Reply:
@dohongdayi I see, the issue can only be reproduced if pushdown is disabled. Could you please update the test case to disable pushdown?
```
  test("count super large table") {
    if (!skipBigDataTest) {
      SnowflakeConnectorUtils.disablePushdownSession(sparkSession)
      val expectedCount = Int.MaxValue + 1L
      val actualCount = sparkSession.read
        .format(SNOWFLAKE_SOURCE_NAME)
        .options(thisConnectorOptionsNoTable)
        .option("autopushdown", "false")
        .option("query", s"select 1 from table(generator(rowcount => $expectedCount))")
        .load()
        .count()
      assert(actualCount == expectedCount)
    }
  }
```